src/issue-creator/issue-reporter.ts

```diff
@@ -1,25 +1,42 @@
-import { HttpService } from "@nestjs/common"
+import { HttpService } from '@nestjs/common';
+import { request, gql } from 'graphql-request'
+import { IssueFormat } from '../IssueFormat';
 
-/*
-  Provides the basic funtionality every "detailed" IssueComponent should have
-*/
+/**
+ * Provides the basic functionality every "detailed" IssueComponent should have
+ */
 export abstract class IssueReporter {
+  api = 'http://localhost:8080/api'; 
 
-    private api = "http://localhost:6969"  // TODO: What API Address
+  constructor(private http: HttpService) { }
 
-    constructor(private http: HttpService) { }
-
-    /**
-      Takes an @param issue and sends Issues to the (not yet implemented) api.
-      Can be called in every Component that extends IssueComponent
-    */
-    // TODO: Create Issue Format (Interface)
-    reportIssue(issue: any) {
-        console.log("Reporting Issue")
-        // TODO: Where/What to send to API
-        this.http.post(this.api, issue).subscribe(
-            res => console.log("Issue Report Successful!"),
-            err => console.log("Error at reporting issue")
-        )
+  /**
+   * Takes an @param issue and sends Issues to the current MockApi and receives the Issue ID
+   * Can be called in every Component that extends IssueComponent.
+   * @returns the issueID received when the the request was accepted by the server
+   */
```

Comment:
wrong usage of tags!!!

```suggestion
  /**
   * sends an Issues to the current MockApi and receives the Issue ID.
   * can be called in every Component that extends IssueComponent.
   *
   * @param issue the issue to be send
   * @returns the issueID received when the the request was accepted by the server
   */
```